ush/config_defaults.sh

```diff
@@ -352,12 +352,20 @@ WFLOW_LAUNCH_LOG_FN="log.launch_FV3LAM_wflow"
 # FCST_LEN_HRS:
 # The length of each forecast, in integer hours.
 #
+# DA_CYCLE_INTERV:
+# Data assimilation cycle interval
+#
+# RESTART_INTERVAL:
+# Set up how often or which forecast hours the restart files should be
+# generated. This need to match with DA_CYCLE_INTERV.
```

Comment:
There's a typo here that makes this unclear. Maybe "Frequency to write restart files from forecast in hours." You should also include information on the type of the value. It looks like a list, but it's unclear on what should be included in the list.

Reply:
Changed to
"+# Set up frequenency or list of the forecast hours that FV3 should
+# generate the restart files. The numbers need to match with DA_CYCLE_INTERV"

Hope this is more clear. It is very trick to setup the restart files for cycling.